discordkit/utils/event.ts

```
@@ -0,0 +1,46 @@
+                  logError(
+                    `Error executing event ${String(config.name)}:`,
+                    error,
+                  );
+                }),
+              ),
+            ),
+          );
+        },
+      };
+    }
+  };
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)